### PR TITLE
Rename fluence sdk to marine-rs-sdk

### PR DESCRIPTION
**Pull request by mikevoronov**

### CODE REVIEW

File: crates/marine-macro-impl/src/parsed_type/utils.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * copyright 2020 fluence labs limited
+ * copyright 2020 sdk labs limited
```
Comment (folex) [inline crates/marine-macro-impl/src/parsed_type/utils.rs:2]: typo

File: crates/main/src/logger.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@
 //! Macros from crate [`log`] are used as a logging facade.
 //!
 //! ```ignore
-//!     use fluence::logger;
+//!     use sdk::logger;
```
Comment (folex) [inline crates/main/src/logger.rs:26]: shouldn't it be `marine_rs_sdk::logger`?

File: crates/main/src/lib.rs [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 #![allow(clippy::missing_safety_doc)]
 #![allow(clippy::needless_doctest_main)]
-#![doc(html_root_url = "https://docs.rs/fluence-sdk-main/0.6.9")]
+#![doc(html_root_url = "https://docs.rs/sdk-sdk-main/0.6.9")]
```
Comment (folex) [inline crates/main/src/lib.rs:22]: `marine-rs-sdk-main/0.6.9` ?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ![crates.io version](https://img.shields.io/crates/v/fluence?color=green)
 
-## Rust backend SDK
+## Marine Rust SDK
 
-This SDK intended to run backend application on the Fluence network. More information about usage and some internals could found in [docs](https://fluence-labs.readme.io/docs/rust-backend-sdk).
+This SDK is intended to run backend application on the Fluence network with [Marine](https://github.com/fluencelabs/marine) runtime. More information about usage and some internals could found in [docs](https://fluence-labs.readme.io/docs/rust-backend-sdk).
```
Comment (folex) [inline README.md:5]: Uh oh. Link to docs currently leads to the `fluence.dev` :( I'll try to fix that ASAP so it points to the old documentation, BUT. 

But maybe let's point to to https://doc.fluence.dev/docs/development_development for example?

**Review by folex — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (folex) [reply README.md:5]: Seems that `https://fluence-labs.readme.io/docs/rust-backend-sdk` is now pointing to the old docs, so link is kinda fine. Ideally though we need to copy paste that doc to the latest documentation

**Comment by boneyard93501** [comment]
done. last week.
https://doc.fluence.dev/docs/knowledge_knowledge/knowledge_aquamarine/marine

**Comment by mikevoronov** [comment]
> done. last week.
> https://doc.fluence.dev/docs/knowledge_knowledge/knowledge_aquamarine/marine

Thank you, @boneyard93501! I'll use https://doc.fluence.dev/docs/knowledge_knowledge/knowledge_aquamarine/marine/marine-rs-sdk in README then.

**Pull request merged**